[2024-04-20 09:56:37,204] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_toll_data.transform_data manual__2024-04-20T09:56:29+00:00 [queued]>
[2024-04-20 09:56:37,218] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_toll_data.transform_data manual__2024-04-20T09:56:29+00:00 [queued]>
[2024-04-20 09:56:37,218] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-04-20 09:56:37,218] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-04-20 09:56:37,219] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-04-20 09:56:37,238] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): transform_data> on 2024-04-20 09:56:29+00:00
[2024-04-20 09:56:37,250] {standard_task_runner.py:52} INFO - Started process 9294 to run task
[2024-04-20 09:56:37,255] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_toll_data', 'transform_data', 'manual__2024-04-20T09:56:29+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/ETL_toll_data.py', '--cfg-path', '/tmp/tmpcdymaayv', '--error-file', '/tmp/tmpv6szswsm']
[2024-04-20 09:56:37,256] {standard_task_runner.py:80} INFO - Job 39: Subtask transform_data
[2024-04-20 09:56:37,369] {task_command.py:371} INFO - Running <TaskInstance: ETL_toll_data.transform_data manual__2024-04-20T09:56:29+00:00 [running]> on host f3257ad4c610
[2024-04-20 09:56:37,492] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=longdvnl@gmail.com
AIRFLOW_CTX_DAG_OWNER=Longdvnl
AIRFLOW_CTX_DAG_ID=ETL_toll_data
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2024-04-20T09:56:29+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-20T09:56:29+00:00
[2024-04-20 09:56:37,493] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-04-20 09:56:37,494] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'tr "[a-z]" "A-Z" < /home/project/***/dags/finalassignment/extracted_data.csv >         /home/project/***/dags/finalassignment/staging/transformed_data.csv']
[2024-04-20 09:56:37,518] {subprocess.py:85} INFO - Output:
[2024-04-20 09:56:37,529] {subprocess.py:96} INFO - Command exited with return code 0
[2024-04-20 09:56:37,591] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_toll_data, task_id=transform_data, execution_date=20240420T095629, start_date=20240420T095637, end_date=20240420T095637
[2024-04-20 09:56:37,627] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-20 09:56:37,691] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
